### PR TITLE
[WIP] Add Bagnowka pipeline (download)

### PR DESCRIPTION
**Pull request by Libisch**
First step of the new pipeline for external source "Bagnowka" (scraped from www.bagnowka.pl).
* Add _download_ prossesor
* Add _convert_ prossesor

### CODE REVIEW

File: datapackage_pipelines_mojp/bagnowka/processors/download.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @classmethod
+    def _get_schema(cls):
+        # TODO change scheme
+        return {"fields": [{"name": "source", "type": "string"}, {"name": "collection", "type": "string"},
+            {"name": "name", "type": "string"},
+            {"name": "main_image_url_aws_s3", "type": "string"},
+            {"name": "description", "type": "string"},
+            {"name": "id", "type": "integer"},
+            {"name": "date_taken", "type": "string"},
+            {"name": "parsed_doc", "type": "object"}]}
```
Comment (OriHoch) [inline datapackage_pipelines_mojp/bagnowka/processors/download.py]: * source and collection fields are not necessary
* main_image_url is enough, no need to specify that it's in aws_s3 in the field name
  * you can provide this detail in the description:
    * `{"name": "main_image_url", "type": "string", "description": "stored in aws S3"}`
* date_taken should be one of the date or year field types - https://pre-v1.frictionlessdata.io/json-table-schema/#date
* parsed_doc shouldn't be needed here, you should be able to have all values available as distinct fields in the output rows

File: datapackage_pipelines_mojp/bagnowka/processors/download.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+    def _get_resource(self):
+        all_items_file = open("/Users/libi/mojp-dbs-pipelines/datapackage_pipelines_mojp/bagnowka/processors/bagnowka_all.json", "r")
```
Comment (OriHoch) [inline datapackage_pipelines_mojp/bagnowka/processors/download.py]: you should open the file like this:
```
with open(`os.path.join(os.path.dirname(__file__), "bagnowka_all.json")`) as f:
    all_docs = json.load(f)
```
this ensures file is closed when you are done reading from it

File: datapackage_pipelines_mojp/bagnowka/processors/download.py (hunk 1 above)
Comment (Libisch) [inline datapackage_pipelines_mojp/bagnowka/processors/download.py]: @OriHoch dates were too ununified, added a detailed description and left the type "string". will try to handle it in convert.py

**Comment by Libisch** [comment]
@OriHoch fixed (besides the [date](https://github.com/Beit-Hatfutsot/mojp-dbs-pipelines/pull/68/files#diff-24446c8d10305dfa16960b0d5bf764f1R23). kept it as string and added a description).

**Pull request merged**